### PR TITLE
[to #223] feat: Let TempFile::WriteBytesToFile return status.

### PR DESCRIPTION
**Pull request by qiuxiafei**

### CODE REVIEW

File: pytorch_blade/src/common_utils/BUILD [hunk 1]
```diff
@@ -28,21 +32,34 @@ cc_library(
...
 
 cc_library(
     name = "torch_blade_common",
     includes = ["../"],
     deps = [
         ":torch_blade_logging",
-        ":torch_blade_tmpfs",
         ":torch_blade_macros",
         ":torch_blade_utils",
+        ":torch_blade_version",
```
Comment (fortianyou) [inline pytorch_blade/src/common_utils/BUILD:49]: `:torch_blade_version` has been removed, please rebase.

File: pytorch_blade/src/common_utils/tempfs.h [hunk 2]
```diff
@@ -14,16 +14,27 @@
...
 namespace blade {
 
 class TempFile {
  public:
   TempFile();
   ~TempFile();
-  void WriteBytesToFile(const std::string& bytes);
+  DISALLOW_COPY_AND_ASSIGN(TempFile);
+
+  /// Write byte content to temp file and return number of bytes written. Return
```
[the remark this answers is not in the document]
Comment (qiuxiafei) [reply pytorch_blade/src/common_utils/tempfs.h]: Fixed ...

File: pytorch_blade/src/common_utils/BUILD (hunk 1 above)
Comment (qiuxiafei) [reply pytorch_blade/src/common_utils/BUILD]: Done.

**Review by fortianyou — APPROVED** [review]
LGTM

**Pull request merged**